src/api/posts.js

```diff
@@ -0,0 +1,30 @@
+import { client } from './../utils/fetchClient';
+
+export const getPosts = async userId => {
+  const response = await client.get('/posts', {
+    params: { userId },
+  });
+
+  return response.data;
+};
+
+export const addPost = async postData => {
+  const response = await client.post(`/posts`, {
+    userId: USER_ID,
```

Comment:
The `USER_ID` variable is used here but is not defined or imported in this file. You need to define it or import it from the appropriate module to ensure this function works correctly.